plonk-wasm/src/pasta_fp_plonk_index.rs

```diff
@@ -271,7 +270,7 @@ pub fn caml_pasta_fp_plonk_index_write(
 #[wasm_bindgen]
 pub fn caml_pasta_fp_plonk_index_serialize(index: &WasmPastaFpPlonkIndex) -> String {
     let serialized = rmp_serde::to_vec(&index.0).unwrap();
-    general_purpose::STANDARD.encode(serialized)
+    base64::encode(serialized)
```

Comment:
Nit: it would be better to rebase the branch and remove this "revert revert revert" change together with "revert revert" change earlier in the same PR to avoid confusion... 

this looks quite innocent to me btw but we can keep it in the old `base64` way for now for clarity.

Reply:
>this looks quite innocent to me btw but we can keep it in the old base64 way for now for clarity

Spoiler: it is not innocent.